public/assets/images/icons/checkbox-icon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="11.3" viewBox="0 0 14 11.3"><polygon fill="#fff" class="cls-1" points="12.2 0 4.8 7.8 1.7 4.7 0 6.5 4.8 11.3 14 1.7 12.2 0"/></svg>
```

Comment:
Polygons shouldn't not have fill nor class attributes :) We had problems with this before